### PR TITLE
[13.0][MIG] hr_personal_equipment_request: Migration to 13.0

### PR DESCRIPTION
**Pull request by albariera**
This module is still not merged into version 12.0. The PR can be found at #1004

### CODE REVIEW

**Review by etobella — APPROVED** [review]
Code Review LGTM
I see it is not accessible from `hr.employee.public`. I don't know if that is necessary, but it seems right for me.

**Comment by etobella** [comment]
@marcelsavegnago

File: hr_personal_equipment_request/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "depends": ["product", "hr", "mail"],
+    "data": [
+        "security/hr_personal_equipment_request_security.xml",
+        "security/ir.model.access.csv",
+        "views/product_template.xml",
+        "views/hr_personal_equipment.xml",
+        "views/hr_personal_equipment_request.xml",
+        "views/hr_employee.xml",
+    ],
+    "demo": [],
```
Comment (MeetKD) [inline hr_personal_equipment_request/__manifest__.py]: Remove empty key if not needed

Comment (albariera) [reply hr_personal_equipment_request/__manifest__.py]: Done! Thanks!

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM! as a suggestion I would at the expiry_date when an allocation expires.

**Comment by albariera** [comment]
Done! @AaronHForgeFlow Thanks for your review! :blush:

**Comment by etobella** [comment]
@marcelsavegnago WDYT?

**Comment by etobella** [comment]
@OCA/human-resources-maintainers

**Pull request merged**